Remove o uso do vega lite

deboraleda opened the pull request:
o grafo da página inicial do app havia sido construído com a biblioteca vega lite mas foi decisão da equipe uniformizar todo o código com o uso do D3 JS, sendo assim, o código em vega foi removido. Esse PR foi feito a partir da branch `init-d3`

deboraleda committed:
adiciona configuração do d3 e funcionalidades principais

deboraleda committed:
adiciona rota para trilha de infra

deboraleda committed:
remove código estrutural

deboraleda committed:
adiciona classe d3Graph

essa classe tem como responsabilidade encapsular todas as funcionalidades
do d3 que são necessárias para gerar o grafo em formato de árvore

deboraleda committed:
adiciona mvp de visualização com diferentes trilhas

deboraleda committed:
adiciona seta nos arcos

deboraleda committed:
remove o uso da biblioteca vega lite

o grafo da página inicial do app havia sido construído com a biblioteca
vega lite mas foi decisão do projeto uniformizar todo o código com o uso do
D3 JS, sendo assim, o código em vega foi removido

deboraleda committed:
remove vega das dependências

LeandraOS commented on src/Components/Home/d3-force-layout.js:
```diff
@@ -0,0 +1,110 @@
+        this.width = 600
+        this.height = 700
+
+
```
Indico retirar excesso de linhas em branco :rocket:

LeandraOS commented on src/Components/Home/miserables.js:
```diff
@@ -0,0 +1,69 @@
+    ]
+}
+
+export default {graphData};
```
Indico adicionar linha em branco ao final do arquivo :rocket:

LeandraOS commented on src/Components/TrilhaInfra/Trilha-infra.jsx:
```diff
@@ -0,0 +1,80 @@
+}
+
+ 
+export default TrilhaInfra;
```
Indico retirar excesso de linhas em branco  e adicionar linha em branco ao final do arquivo :rocket:

LeandraOS commented on src/Components/TrilhaInfra/d3-graph.js:
```diff
@@ -0,0 +1,224 @@
+}
+
+export default GraphD3
+  
```
Indico retirar excesso de linhas em branco e adicionar linha em branco ao final do arquivo :rocket:

LeandraOS commented on src/Components/Home/d3-force-layout.js:
```diff
@@ -0,0 +1,110 @@
+            .data(graphData.links)
+            .enter().append("line")
+            .attr("stroke-width", function(d) { return 1.5 })
+            .attr("stroke", "#999");
```
Indico alterar essa cor das arestas, ela ficou bem pesada. 
Podemos trocar pela `#ccc`.

Além disso, acredito que mais algumas alterações são necessárias:

#### Nos nodes:

-  eles podem estar mais espaçados
- e menores

#### No grafo em si: 

-  ele pode estar com um efeito de rotação bem semelhante ao anterior feito no vega
- efeito de movimentação(confusão) também semelhante ao anterior feito no vega


> Você acha que é possível implementar essas mudanças?

deboraleda committed:
adiciona movimento de rotação

foi solicitado que o grafo fique sempre com o movimento de rotação,
isso foi feito setando valores para os parâmetros alphaDecay e
alphaTarget, como sugere a documentação do D3. Além disso,
as linhas em branco dos arquivos foram removidas

deboraleda committed:
aumenta força de repulsão entre os nós do grafo

LeandraOS committed:
Merge branch 'v2-testes' into change-vega-to-d3
